tools/advisor/test/input_files/test_rules.ini

```diff
@@ -17,22 +17,18 @@ conditions=log-4-false:options-1-false
 [Condition "log-1-true"]
 source=LOG
 regex=Stopping writes because we have \d+ immutable memtables \(waiting for flush\), max_write_buffer_number is set to \d+
-scope=column_family
 
 [Condition "log-2-true"]
 source=LOG
 regex=Stalling writes because we have \d+ level-0 files
-scope=column_family
```

Comment:
Why did we remove the scope keyword?

Reply:
I thought that the code should be able to figure out the scope on its own when the triggers are set for the conditions.